central/globaldb/export/backup.go

```diff
@@ -42,3 +43,15 @@ func Backup(ctx context.Context, boltDB *bolt.DB, rocksDB *rocksdb.RocksDB, incl
 
 	return zipWriter.Close()
 }
+
+// BackupPostgres backs up the given databases (optionally removing secrets) and writes a ZIP archive to the given writer.
+func BackupPostgres(ctx context.Context, postgresDB *pgxpool.Pool, includeCerts bool, out io.Writer) error {
```

Comment:
includeCerts is not used? We still need to backup all the certs and version file etc.

Reply:
correct, BUT when I split up the story into parts I put certs in ROX-10087 so I could keep the work focused